README.md

```diff
@@ -161,7 +163,7 @@ Name | Description
 [community.aws.iam_role](https://github.com/ansible-collections/community.aws/blob/main/docs/community.aws.iam_role_module.rst)|Manage AWS IAM roles
 [community.aws.iam_role_info](https://github.com/ansible-collections/community.aws/blob/main/docs/community.aws.iam_role_info_module.rst)|Gather information on IAM roles
 [community.aws.iam_saml_federation](https://github.com/ansible-collections/community.aws/blob/main/docs/community.aws.iam_saml_federation_module.rst)|Maintain IAM SAML federation configuration.
-[community.aws.iam_server_certificate](https://github.com/ansible-collections/community.aws/blob/main/docs/community.aws.iam_server_certificate_module.rst)|Manage server certificates for use on ELBs and CloudFront
+[community.aws.iam_server_certificate](https://github.com/ansible-collections/community.aws/blob/main/docs/community.aws.iam_server_certificate_module.rst)|Manage IAM server certificates for use on ELBs and CloudFront
```

Comment:
Why is here this change?

Reply:
The short description was updated in #1276 (because there's also ACM for cert management) but I didn't re-run the docs generation as part of that PR, so it wasn't updated in README.md

As part of these moves I've been running `collection_prep_add_docs` to ensure docs get moved about.  Mostly I've been excluding unrelated changes to the generated docs.

The changed description is in the module itself, and this part of README.md is autogenerated based off the docs.